docs/src/test-sharding-js.md

```diff
@@ -60,7 +60,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        shard: [1, 2, 3, 4]
+        shard: [1/4, 2/4, 3/4, 4/4]
```

Comment:
Do you mind adjusting these two places to the new system as well? So they are aligned:

- https://playwright.dev/docs/ci#on-pushpull_request-sharded
- https://playwright.dev/docs/ci#via-containers-sharded